BraveWallet/Crypto/Transaction Confirmations/TransactionConfirmationView.swift

```diff
@@ -235,6 +235,24 @@ struct TransactionConfirmationView: View {
                   }
                   .padding()
                   .accessibilityElement(children: .contain)
+                  Divider()
+                    .padding(.leading)
+                  NavigationLink(destination: EditNonceView(
+                    confirmationStore: confirmationStore,
+                    transaction: activeTransaction
+                  )) {
```

Comment:
Is custom nonce only available for 1559 transactions? At https://github.com/brave/brave-ios/pull/5078/files#diff-a30f2b87987cc8c044dccfca6eeeeebce278d6e1d02a0d10c6c4630f16c32495R62 you are specifically pulling nonce out of `txDataUnion.ethTxData1559`. Do you need to pull it from regular `ethTxData` for non-1559 txs?